retroshare-gui/src/gui/feeds/GxsChannelPostItem.cpp

```diff
@@ -389,12 +389,26 @@ void GxsChannelPostItem::fill()
 	mInFill = true;
 
 	QString title;
+	
+	float f = QFontMetricsF(font()).height()/14.0 ;
 
 	if(mPost.mThumbnail.mData != NULL)
 	{
-		QPixmap thumbnail;
+		QPixmap thumbnail;	
+		
 		GxsIdDetails::loadPixmapFromData(mPost.mThumbnail.mData, mPost.mThumbnail.mSize, thumbnail,GxsIdDetails::ORIGINAL);
 		// Wiping data - as its been passed to thumbnail.
+		if( thumbnail.width() < 90 ){
+			ui->logoLabel->setMaximumSize(82,108*f);
+		}
+		else if( thumbnail.width() < 109 ){
+			ui->logoLabel->setMinimumSize(108,108*f);
+			ui->logoLabel->setMaximumSize(108,108*f);
+		}
+		else{
+			ui->logoLabel->setMinimumSize(156,108*f);
```

Comment:
It seems to me that you need to use f in both left and right, because the image should be scaled both ways. I mean:
setMinimumSize(108*f,108*f)

Reply:
ok i will change

Reply:
done